card/mod/05_standard/spec/set/type/image_spec.rb

```diff
@@ -8,7 +8,7 @@
     end
   end
 
-  it "handles size argument in inclusion syntax" do
+  it "handles size argument in nest syntax" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.